File: app/src/androidTest/java/com/osorio/junior/laboratoriocalificado02/MainActivity.java
package com.osorio.junior.laboratoriocalificado02;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button openRegistroButton = findViewById(R.id.openRegistroPedidosButton);
        openRegistroButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, RegistroPedidosActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/osorio/junior/laboratoriocalificado02/MainActivity.kt
package com.osorio.junior.laboratoriocalificado02

import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_ejercicio01)  // Usamos el layout activity_ejercicio01.xml directamente

        // Inicializa los botones y el View verde usando findViewById
        val greenView = findViewById<View>(R.id.greenView)
        val showButton = findViewById<Button>(R.id.showButton)
        val hideButton = findViewById<Button>(R.id.hideButton)

        // Inicialmente, el View verde está oculto
        greenView.visibility = View.GONE

        // Botón Mostrar: Hace visible el View verde
        showButton.setOnClickListener {
            greenView.visibility = View.VISIBLE
        }

        // Botón Ocultar: Oculta el View verde
        hideButton.setOnClickListener {
            greenView.visibility = View.GONE
        }
    }
}
